### PR TITLE
chore: use the "files" field

### PR DESCRIPTION
**Pull request by evilebottnawi**
**What kind of change does this PR introduce?**

chore

**Did you add tests for your changes?**

No need

**If relevant, did you update the documentation?**

No need

**Summary**

Simplify

**Does this PR introduce a breaking change?**

No

**Other information**

No

### CODE REVIEW

File: packages/webpack-cli/package.json [hunk 1]
```diff
@@ -22,6 +22,9 @@
     "bundler",
     "web"
   ],
+  "files": [
+    "lib"
```
Comment (anshumanv) [inline packages/webpack-cli/package.json:26]: also need `bin`

Comment (evilebottnawi) [reply packages/webpack-cli/package.json:26]: Nope, we have the `"bin"` field, what mean automatically adding "bin" directories to packages, try to run `npm pack`/`yarn pack`

Comment (evilebottnawi) [reply packages/webpack-cli/package.json:26]: Bug on rule side :smile:

Comment (evilebottnawi) [reply packages/webpack-cli/package.json:26]: Let's adding it here and I will report about this bug

Comment (anshumanv) [reply packages/webpack-cli/package.json:26]: 👍

**Review by anshumanv — DISMISSED** [review]

**Review by snitin315 — APPROVED** [review]

**Review by anshumanv — APPROVED** [review]

**Pull request merged**